experimenter/experimenter/nimbus-ui/src/components/PageSummary/index.tsx

```diff
@@ -398,6 +398,14 @@ const WarningList = ({
       ?.toString()
       .replaceAll(",", ", ");
 
+  const liveExperimentsInNamespace = experiment.liveExperimentsInNamespace
+    ?.toString()
+    .replaceAll(",", ", ");
```

Comment:
It could be an empty string `""` once it's .join()ed or .toStringed

Reply:
The graphql stuff is a bit wonky because it's a query. It is a non-null field, but it still needs the `?` becuase it's a query. I need to find better answers for _why_, though...